locust/argument_parser.py

```diff
@@ -163,9 +163,8 @@ def download_file_from_url(url: str) -> str:
         sys.stderr.write(f"Failed to get locustfile from: {url}. Exception: {e}")
         sys.exit(1)
 
-    if response.ok:
-        with open(f"{tempfile.gettempdir()}/{url.rsplit('/', 1)[-1]}", "w") as locustfile:
-            locustfile.write(response.text)
+    with open(f"{tempfile.gettempdir()}/{url.rsplit('/', 1)[-1]}", "w") as locustfile:
```

Comment:
I think building the path can be expressed nicer using something like 

`os.path.join(empfile.gettempdir(), url.rsplit('/', 1)[-1])`

Reply:
I'll fix this in master :)